src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -791,31 +797,33 @@ void StateControllerImpl::OnApplicationRegistered(
   OnStateChanged(app, initial_state, new_state);
 }
 
-int64_t StateControllerImpl::SendBCActivateApp(
+int64_t StateControllerImpl::SendStateToHMI(
```

Comment:
@jacobkeeler I would rename it to something like RequestStateChangeToHMI

Reply:
@LuxoftAKutsan fixed with https://github.com/smartdevicelink/sdl_core/pull/2948/commits/8080f10bd7b7f5bbe074cbdafdc19fba7ebe8118